src/api/spec/models/user_spec.rb

```diff
@@ -870,4 +870,23 @@
       end
     end
   end
+
+  describe '#autocomplete' do
+    let!(:foobar) { create(:confirmed_user, login: 'foobar') }
```

Comment:
[nitpick] what about using something that seems to be a login? like a real name? :smile:

Reply:
because its a login and not real name, even described in RFCs https://www.ietf.org/rfc/rfc3092.txt :)